src/PrestaShopBundle/DependencyInjection/Compiler/LoadServicesFromModulesPass.php

```diff
@@ -37,6 +37,8 @@
 
 /**
  * Load services stored in installed modules.
+ *
+ * @deprecated since 9.0, to be removed in 10.0
```

Comment:
```suggestion
 * @deprecated since 9.0, not used anymore to be removed in 10.0
```

Reply:
I hesitate to remove this right away, it's not like it's a class that has chances to have been used or extended outside of the core 🤔